pkg/gcv/validator_test.go

```diff
@@ -221,74 +227,139 @@ func TestAudit(t *testing.T) {
 	}
 }
 
-func TestConvertResourceToInterface(t *testing.T) {
```

Comment:
Not sure we want to lose these tests?

Reply:
This one got relocated to another package

Reply:
Just realized, when I moved this test to the correct package, I forgot to delete it from this one.

Reply:
see https://github.com/forseti-security/config-validator/blob/master/pkg/asset/asset_test.go#L12